content/network/endpoints/index.yaml

```diff
@@ -31,22 +31,14 @@ items:
       rpcUrl:
         name: RPC URL
     items:
-      Ethercluster:
-        __rpcUrl: https://www.ethercluster.com/etc
-        __link: https://www.ethercluster.com
-        __name: Ethercluster
-      Ethercluster (Kotti):
-        __rpcUrl: https://www.ethercluster.com/kotti
-        __link: https://www.ethercluster.com
-        __name: Ethercluster (Kotti)
+      Rivet:
+        __rpcUrl: https://etc.rivet.link
+        __link: https://rivet.link
+        __name: Rivet
       Ethercluster (Mordor):
         __rpcUrl: https://www.ethercluster.com/mordor
```

Comment:
This is correct.  Mordor ONLY at existing URL for now, though we will add another URL elsewhere later (probably under etccooperative.org) so that all of ethercluster.com can go away at the same time.

Reply:
👍